server/server/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@anticrm/server",
   "version": "0.4.0",
-  "main": "lib/index.js",
+  "main": "src/index.ts",
```

Comment:
Why?

Reply:
￼￼￼
Server is not used by its own, so it doesn't really matter for now, it just faster to develop server and start dev/server